api/src/main/java/org/datacleaner/descriptors/DescriptorProvider.java

```diff
@@ -185,6 +180,11 @@
             Class<? extends RenderingFormat<?>> renderingFormat);
 
     /**
+     * Notify all listeners
+     */
+    public void notifyListeners();
```

Comment:
Hmm... Something which has listeners should never have a method to notify them. The whole point of having a listener is that the listener is automatically invoked when it is relevant. It should never be the responsibility of someone coming from the outside and telling that listeners should be notified.